packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.js

```diff
@@ -61,6 +61,14 @@ function findIndex(array, comp) {
   return -1;
 }
 
+function usePrevious(value) {
```

Comment:
I wasn't 100% on what to do with this.

This exact function exists in TreeView, but isn't exported.
We also have usePreviousProps in the utils package.

Reply:
Please use the `usePreviousProps` from the utils package.